application/helpers/admin/import_helper.php

```diff
@@ -1272,6 +1268,30 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
         if (isset($insertdata['tokenlength']) && $insertdata['tokenlength'] > 35) {
             $insertdata['tokenlength'] = 35;
         }
+
+        // Fix survey group when not copying
+        if (!$isCopying) {
+            // Set to 1 by default
```

Comment:
The body of this if-statement could maybe be factored out to a separate function or method, and then that function could be unit-tested.